src/lib/ShowCard.svelte

```diff
@@ -70,8 +70,7 @@
 			<p class="date" style:--transition-name="show-date-{show.number}">
 				{format_show_type(show.date)}
 				<span aria-hidden="true">×</span>
-				<time datetime={show_date.toDateString()} title={show_date.toDateString()}
```

Comment:
Could we keep the title as well for the tooltip? Or does this already do that?

Reply:
Hi Wes, sorry it doesn’t have the tooltip so I can add it back if you want? I am so used to removing it on my own projects so that might have been why.

I see the title attribute was mentioned in a post on the [time element on CSS-Tricks by Chris Coyier](https://css-tricks.com/time-element/) back in 2014, but this ma introduce some accessibility challenges.

I can raise the following in another issue if you want to discuss it. When you use the title attribute like this for a hover tooltip it is providing an inaccessible experience.

<img width="192" alt="title html element saying yesterday with a title tooltip Friday March 1st 2024" src="https://github.com/syntaxfm/website/assets/12486130/3a070589-588b-4461-8883-39f6d34a21f9">

- The title attribute has mixed browser support so doesn't get displayed consistently
- It excludes mobile users
- It excludes keyboard only users

In some cases the title attribute might be used for calculating an accessible name, but here it is not the case so it doesn't matter. For example if something had no accessible name but had a title it might use it.

Scott O’Hara and Sarah Higley wrote about creating accessible tooltips if that is something you’d like to explore?

- [Tooltips in the time of WCAG 2.1 | Sarah Higley](https://sarahmhigley.com/writing/tooltips-in-wcag-21/#best-practices-summary)
- [GitHub - scottaohara/a11y_tooltips: Accessible Tooltip Component](https://github.com/scottaohara/a11y_tooltips)

On [syntax.fm/shows](http://syntax.fm/shows) I can see that it says “n weeks ago” and then after a month it says the date. So after 3 weeks the tooltip is kind of redundant.

<img width="184" alt="title html element saying august 9th 2017 and the same hover title tooltip" src="https://github.com/syntaxfm/website/assets/12486130/af82a09e-88c8-412e-a879-d53f50b4a674">

Another suggestion could be to format the date string relative to the users locale and not just ‘en’.
`const timeFormatter = new Intl.RelativeTimeFormat('en', { numeric: 'auto' });`

Reply:
I say we add it back in - it's not inaccessible as is, but the title adds additinoal context on desktop - I use it a lot myself. In this github thread for example:

<img width="376" alt="Screenshot 2024-03-18 at 3 15 05 PM" src="https://github.com/syntaxfm/website/assets/176013/87041619-a560-42cc-8381-f814589914a4">